packages/thirdweb/src/wallets/in-app/web/lib/auth/index.ts

```diff
@@ -152,11 +152,17 @@ export async function authenticate(
         client: ThirdwebClient;
         ecosystem?: Ecosystem;
         redirect: boolean;
+        redirectUrl?: string;
```

Comment:
Note that `redirectUrl` actually exists on `AuthArgsType` under `SingleStepAuthArgsType`

Reply:
I am actually incorrect here —

```ts
{
      strategy: SocialAuthOption;
      openedWindow?: Window;
      closeOpenedWindow?: (window: Window) => void;
      redirectUrl?: string;
}
```

as an option for `SingleStepAuthArgsType`. If `client: ThirdwebClient` is specified from:

```ts
{
        strategy: SocialAuthOption;
        client: ThirdwebClient;
        ecosystem?: Ecosystem;
        redirect: boolean;
}
```

then `redirectUrl` is not included.

 It'll throw the error:
 ```
Types of property 'redirectUrl' are incompatible.
    Type 'string | undefined' is not assignable to type 'undefined'.
      Type 'string' is not assignable to type 'undefined'
```